translations/en.yaml

```diff
@@ -463,6 +463,18 @@
 "TeamUpdate::DESCRIPTION": Updates the name of your Team.
 "TeamUpdate::NAME": The name of your Team.
 
+"TeamMembers::SUMMARY": List Team Members
+"TeamMembers::DESCRIPTION": Provides a paginated list of members that belong to a given team.
+"TeamMembers::TEAM_ID": The id of the parent team.
```

Comment:
The context of what parent means here isn't clear. Can you be more specific about what team_id is doing here?
"The id of the team that a member list is being requested from."

Reply:
done